controls/radheatmap/tooltip.md

```diff
@@ -0,0 +1,148 @@
+---
+title: ToolTip
+page_title: ToolTip
+description: ToolTip
+slug: radheatmap-tooltip
+tags: tooltip, heatmap, tooltipcell, celltooltip, heatmaptooltipsettings
+published: True
+position: 5
+---
+
+# ToolTip
+
+The RadHeatMap control provides a tooltip behavior which can be used to visualize arbitrary information related to a data point. The ToolTip functionality is disable by default. To enable it, you need to set __ShowToolTips__ property of the RadHeatMap to true. 
+
+#### __[XAML] Example 1: Enabling the ToolTip behavior of the RadHeatMap__
+
+{{region radheatmap-tooltip_0}}
+	 <telerik:RadHeatMap ShowToolTips="True" />
+{{endregion}}
+
+## Customizing the Tooltip
+
+To customize the apperance of the tooltip you can use the __HeatMapToolTipSettings__ property. For demonstration purposes we will use a simple class that will represent one cell of the RadHeatMap control. This class will expose three properties:
```

Comment:
tell what is the class that holds this property. For example, "you can use the HeatMapToolTipSettings property of RadHeatMap"